views/deliveryEmployee/deleteSuccess.njk

```diff
@@ -0,0 +1,25 @@
+{% extends "templates/layout.njk" %}
+
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/panel/macro.njk" import govukPanel %}
+
+{% block content %}
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      {{ govukPanel({
+        titleText: "Delivery Employee deleted"
+      }) }}
+
+      <p class="govuk-body">You have successfully deleted the delivery employee requested.</p>
+
```

Comment:
It would be good if we could tell the user who the employee deleted was here instead of a generic deleted message